langroid/embedding_models/models.py

```diff
@@ -23,6 +25,22 @@ class OpenAIEmbeddingsConfig(EmbeddingModelsConfig):
     context_length: int = 8192
 
 
+class AzureOpenAIEmbeddingsConfig(EmbeddingModelsConfig):
+    model_type: str = "azure-openai"
+    model_name: str = "text-embedding-ada-002"
+    api_key: str = ""
+    azure_endpoint: str = ""
+    azure_deployment: Optional[str] = None
```

Comment:
can you please change it to "deployment_name" - to align with the AzureConfig class?

Reply:
Same here, let's align with AzureConfig's existing names

Reply:
please change name as suggested